x-pack/plugins/actions/server/usage/actions_telemetry.ts

```diff
@@ -26,7 +26,12 @@ export async function getTotalCount(
       init_script: 'state.types = [:]',
       map_script: `
         String actionType = doc['action.actionTypeId'].value;
-        state.types.put(actionType, state.types.containsKey(actionType) ? state.types.get(actionType) + 1 : 1);
+        if (actionType =~ /.gen-ai/) {
+          String genAiActionType = actionType +"__"+ doc['action.config.apiProvider'].value;
```

Comment:
I assume this is fine because generally only humans consume this data (via searches / dashes), but it is a little weird that we're essentially recording a different connector type than what it actually is.  Could be confusing to users.  Also, it means you won't get a combined count of the types, at least automatically.

Reply:
TBH I was trying to avoid adding new fields because this PR/repo intimidates me: https://github.com/elastic/telemetry/pull/1427

I think I might have to touch it anyways since I am adding new fields. I'll take a closer look